packages/aws-cdk/lib/notices.ts

```diff
@@ -36,13 +36,6 @@ export interface NoticesProps {
    */
   readonly output?: string;
 
-  /**
-   * Global CLI option for whether we show notices
-   *
-   * @default true
-   */
-  readonly shouldDisplay?: boolean;
```

Comment:
Messages are now always emitted -- whether they are displayed or not is up to the IoHost.